controlplane/kubeadm/internal/workload_cluster_conditions.go

```diff
@@ -190,6 +292,13 @@ func (w *Workload) getCurrentEtcdMembers(ctx context.Context, machine *clusterv1
 		// NB. We should never be in here, given that we just received answer to the etcd calls included in forFirstAvailableNode;
 		// however, we are considering the calls to Members a signal of etcd not being stable.
 		conditions.MarkFalse(machine, controlplanev1.MachineEtcdMemberHealthyCondition, controlplanev1.EtcdMemberUnhealthyReason, clusterv1.ConditionSeverityError, "Failed get answer from the etcd member on the %s node", nodeName)
+
+		v1beta2conditions.Set(machine, metav1.Condition{
+			Type:    controlplanev1.KubeadmControlPlaneMachineEtcdMemberHealthyV1Beta2Condition,
+			Status:  metav1.ConditionUnknown,
```

Comment:
Is this intentionally unknown? Just asking because we used False before (but it seems okay to me to use unknown in this case)

Reply:
Yes, it is intentional.
With the new condition are using consistently unknown where we cannot determine status of the underlying resource